Rakefile

```diff
@@ -9,7 +9,8 @@ namespace :generate do
 
   task :cucumber do
     cucumber = "#{Gem.loaded_specs['cucumber'].gem_dir}/bin/cucumber"
-    run_ruby_acceptance "-rci/reporter/rake/cucumber_loader -S #{cucumber} --format CI::Reporter::Cucumber acceptance/cucumber"
+    # run_ruby_acceptance "-rci/reporter/rake/cucumber_loader -S #{cucumber} --format CI::Reporter::Cucumber acceptance/cucumber"
+    run_ruby_acceptance "-rci/reporter/rake/cucumber_loader -S #{cucumber} --format CI::Reporter::Cucumber"
```

Comment:
Instead of moving the directory, I'd rather add `-r acceptance/cucumber/step_definitions/development_steps.rb` to this command.